context/DocsContext.tsx

```diff
@@ -1,8 +1,21 @@
 import { createContext } from 'react';
 import { DocsContextType } from '@/types/context/DocsContext';
 
+const initialPost = {
+  title: "",
+  weight: 0,
+  toc: [],
+  readingTime: 0,
+  excerpt: "",
+  sectionSlug: "",
+  sectionWeight: 0,
+  id: "",
+  isIndex: true,
+  slug: ""
+};
+
 const initialDocsContext: DocsContextType = {
-  post: [],
+  post: initialPost,
```

Comment:
Use Post type here from `types/post.ts`.

Reply:
it is using type from post.ts in (DocsContextType file)